src/lib/viewers/BaseViewer.js

```diff
@@ -264,6 +264,10 @@ class BaseViewer extends EventEmitter {
         if (this.containerEl) {
             this.containerEl.removeEventListener('contextmenu', this.preventDefault);
             this.containerEl.innerHTML = '';
+
+            if (this.options.enableAnnotationsDiscoverability) {
```

Comment:
Is this guard clause needed?

Reply:
I think it wouldn't hurt to attempt to remove `'bp-annotations-discoverable'` even if the FF was off so this guard clause isn't needed. Would be open to removing it

Reply:
Discussed offline, will make code simpler so will remove it.